### PR TITLE
Add the  image-repository  address of the container in keadm init/upgrade.

### PR DESCRIPTION
**Pull request by luomengY**
Change the print helm chart resources YAML command name keadm generate to keadm manifest.

Add the  image-repository  address of the container in keadm init/upgrade.

**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
image

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubeedge/kubeedge/issues/5317

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/helm/cloudcore.go [hunk 1]
```diff
@@ -66,6 +66,7 @@ const (
 )
 
 var setsKeyImageTags = []string{"cloudCore.image.tag", "iptablesManager.image.tag", "controllerManager.image.tag"}
+var setsKeyImageRepositories = map[string]string{"cloudCore.image.repository": "kubeedge/cloudcore", "iptablesManager.image.repository": "kubeedge/iptables-manager", "controllerManager.image.repository": "kubeedge/controller-manager"}
```
Comment (WillardHu) [inline keadm/cmd/keadm/app/cmd/helm/cloudcore.go:69]: This is the best flag imageRepository format: `{registry}/{space}` (e.g. my.harbor.cn/my-kubeedge), this allows more flexibility to customize the image.
```golang
var setsKeyImageRepositoriesAndNameMapper = map[string]string{
    "cloudCore.image.repository": "cloudcore", 
    "iptablesManager.image.repository": "iptables-manager", 
    "controllerManager.image.repository": "controller-manager"
}

for k, v := range setsKeyImageRepositories {
    sets = append(opts.Sets, fmt.Sprintf("%s=%s", k, opts.ImageRepository+"/"+v))
}

```

Comment (luomengY) [reply keadm/cmd/keadm/app/cmd/helm/cloudcore.go]: ok

**Comment by luomengY** [comment]
add to v1.17.@Shelley-BaoYue

**Review by fisherxu — COMMENTED** [review]
/approve
/assign @WillardHu

**Pull request merged**